src/datajudge/requirements.py

```diff
@@ -270,17 +275,30 @@ def add_uniques_equality_constraint(
         of a ``DataSource`` are strictly the ones of a reference set of expected values,
         specified via the ``uniques`` parameter.
 
+        Null values in the column are ignored. To assert the non-existence of them use
+        the ``NullAbsence`` constraint via the ``add_null_absence_constraint`` helper method
+        for ``WithinRequirement``.
+        By default, the null filtering does not trigger if multiple columns are fetched at once.
+        It can be configured in more detail by supplying a custom ``filter_func`` function.
+        Some exemplary implementations are available in this module as ``datajudge.utils.filternull_element``,
+        ``datajudge.utils.filternull_never``, ``datajudge.utils.filternull_element_or_tuple_all``, ``datajudge.utils.filternull_element_or_tuple_any``.
+        Passing None as the argument is equivalent to ``datajudge.utils.filternull_element``, but triggers a warning.
+        The deprecated default may change in future versions.
```

Comment:
What is meant by that? That the deprecated setup will be removed or that a different default might be deprecated or that the default will become the non-deprecated option?

Reply:
In-depth explanation added.